apps/prairielearn/assets/scripts/navbarClient.ts

```diff
@@ -144,38 +144,38 @@ onDocumentReady(() => {
     }
   });
 
-  const navbarUserEffectiveUidInput = document.querySelector<HTMLInputElement>(
-    '#navbar-user-effective-uid-input',
-  );
-  const navbarUserEffectiveUidButton = document.querySelector<HTMLButtonElement>(
-    '#navbar-user-effective-uid-button',
-  );
-
-  navbarUserEffectiveUidInput?.addEventListener('keyup', () => {
-    if (navbarUserEffectiveUidInput.value.trim() !== '') {
-      navbarUserEffectiveUidButton.removeAttribute('disabled');
+  document.querySelectorAll<HTMLButtonElement>('.js-remove-override').forEach((element) => {
+    element.addEventListener('click', () => {
+      const cookieName = element.dataset.overrideCookie;
+      Cookies.remove(cookieName, { path: '/' });
+      location.reload();
+    });
+  });
+
+  const effectiveUidInput = document.querySelector<HTMLInputElement>('.js-effective-uid-input');
+  const effectiveUidButton = document.querySelector<HTMLButtonElement>('.js-effective-uid-button');
+
+  effectiveUidInput.addEventListener('input', () => {
```

Comment:
This may be null (e.g., if logged in user is a student).
```suggestion
  effectiveUidInput?.addEventListener('input', () => {
```

Reply:
Great call, resolved in 5c99d61.